tools/tcg_eventlog_tool/build.gradle

```diff
@@ -26,6 +27,26 @@ dependencies {
 
 ext.configDir = new File(projectDir, 'config')
 
+checkstyle {
+    toolVersion = '10.12.7'
```

Comment:
Do you need to specify the version number if it's already in settings.gradle?

Reply:
I believe the settings.gradle line can be removed.  I will test and repush.